QiitaForMe/QiitaForMe/QiitaItemViewController.swift

```diff
@@ -22,24 +26,84 @@ class QiitaItemViewController: UIViewController {
         // Do any additional setup after loading the view.
         qiitaItemWebView.uiDelegate = self
         qiitaItemWebView.navigationDelegate = self
+        
+        // スワイプで履歴を戻る／進むを有効化
+        qiitaItemWebView.allowsBackForwardNavigationGestures = true
+        
+        // navigationControllerのスワイプジェスチャー制御用
+        navigationController?.interactivePopGestureRecognizer?.delegate = self
+        
+        // Pull to Refresh の設定
+        qiitaItemWebView.scrollView.bounces = true
+        qiitaItemWebView.scrollView.refreshControl = refreshControl
+        refreshControl.addTarget(self, action: #selector(QiitaItemViewController.refresh(_:)), for: .valueChanged)
```

Comment:
```suggestion
        refreshControl.addTarget(self, action: #selector(refresh), for: .valueChanged)
```
swift4であればselectorはこうやってシンプルに書けると思うのでお試しくださいませ